### ISSUE TITLE
Accessibility: Content of status cell in test queue does not have correct semantic structure

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#1426]
In the test queue in a run table, When there are multiple status phrases in the status column, screen readers will read them as a single phrase. That is, they are concatenated as inline elements instead of block elements.

Each separate status phrase either needs to be in a `div` instead of a `span`, or the separate status phrases need to be structured as list items.

For example, here is how JAWS renders the status cell content when the status cell contains both a status phrase summarizing overall tester status and a status phrase for a bot run.

>0% complete by 3 testers Bot Status:
>list of 2 items
>1 Test Completed
>0 Tests Queued
>list end

Note how the first line:

>0% complete by 3 testers Bot Status:

Combines the overall status phrase with the phrase indicating the start of bot status.

The structure should be:

>* 0% complete by 3 testers
>* Bot Status:
>   * 1 Test Completed
>   * 0 Tests Queued

![Image](https://github.com/user-attachments/assets/d42d637a-87fd-4e94-897e-0799aad4bb20)
